libs/blocks/media/media.js

```diff
@@ -46,7 +54,13 @@ export default function init(el) {
       decorateBlockText(text, blockTypeSizes[size]);
     }
     const image = row.querySelector(':scope > div:not([class])');
-    if (image) image.classList.add('image');
+    const video = row.querySelector(':scope > div > a[href$=".mp4"]');
+    if (video) {
+      decorateVideo(video.closest('div'));
+      image.classList.add('ambient-video');
+    } else {
```

Comment:
To keep current functionality, we would want to change this to an `else if (image) {...` since on line 49 we check for the image first before modifying `classList`